disconnect-blacklist.json

```diff
@@ -10967,6 +10959,7 @@
       {
         "Facebook": {
           "http://www.facebook.com/": [
+            "apps.fbsbx.com",
```

Comment:
`ValueError: Blocklist contains block rule apps.fbsbx.com under the Disconnect category, but the rule is not found in the given Disconnect mapping file.`

Updating ...

Reply:
@englehardt https://github.com/mozilla-services/shavar-list-creation/pull/101